status/cpu_mem.sh

```diff
@@ -0,0 +1,14 @@
+# Requires https://github.com/thewtex/tmux-mem-cpu-load
+
+show_cpu_mem() {
+  local index icon color text module
+
+  index=$1
+  icon="$(get_tmux_option "@catppuccin_cpu_mem_icon" "")"
+  color="$(get_tmux_option "@catppuccin_cpu_mem_color" "$thm_blue")"
+  text="$(get_tmux_option "@catppuccin_cpu_mem__text" "#($TMUX_PLUGIN_MANAGER_PATH/tmux-mem-cpu-load/tmux-mem-cpu-load --cpu-mode 2 --averages-count 0 --interval 2)")"
```

Comment:
```suggestion
  text="$(get_tmux_option "@catppuccin_cpu_mem_text" "#($TMUX_PLUGIN_MANAGER_PATH/tmux-mem-cpu-load/tmux-mem-cpu-load --cpu-mode 2 --averages-count 0 --interval 2)")"
```
2 underscores between mem and text